### ISSUE TITLE
Set up CI tests to run on Windows

### ISSUE THREAD

**Issue by bjankord** [cerner/terra-core#976]
# Issue Description
To help ensure the project can be built on Windows, we need to setup CI tooling that runs on Windows.

## Issue Type

- [ ] New Feature
- [x] Enhancement
- [ ] Bug
- [ ] Other

## Expected Behavior
Tests are run on Windows in addition to current Travis CI test on every pull request

## Current Behavior
Tests are run in Travis CI which is configured to use linux as OS for running tests.

**Comment by bjankord** [comment]
[Appveyor](https://www.appveyor.com/) looks like a popular solution for running CI tests on Windows. 

Here are the instructions for setting it up with a GitHub org: https://www.appveyor.com/docs/team-setup/#setting-up-appveyor-account-for-github-organization

**Comment by bjankord** [comment]
Reading through the documented instructions, they aren't 100% clear, I've found [this thread](https://github.com/appveyor/ci/issues/801#issuecomment-283648700) which offered the most clarity on how to set this up for the Cerner org.

tl;dr version of that thread:  We'll want to create a new GitHub user, give that user access to only the Cerner GitHub org, and use that user's account to authorize AppVeyor.

**Comment by BenBoersma** [comment]
Found this that may suggest that getting docker to work in a Windows VM might be more work then suspected. https://docs.docker.com/docker-for-windows/troubleshoot/#running-docker-for-windows-in-nested-virtualization-scenarios

**Comment by bjankord** [comment]
Ok that sounds good @BenBoersma, thanks for looking into this issue. I agree, it sounds like the nesting issue is to tricky to get this working as we need it to.
